File: src/app/review-dto.ts
export class ReviewDTO {

    id : number;
    hotelId:number;
    roomId:number;
    starRating: number;
	userReviewMsge : string;

    constructor( 
       
    id : number,
    hotelId:number,
    roomId:number,
	starRating : number,
	userReviewMsge : string,

    ){

        this.id=id;
        this.hotelId=hotelId;
        this.roomId=roomId;
        this.starRating=starRating;
        this.userReviewMsge=userReviewMsge;
       
    }
}

File: main/java/com/cg/hbm/dto/review/ReviewDTO.java
package com.cg.hbm.dto.review;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor

public class ReviewDTO {
	
	private int id;
	private String userName;
	
	private int starRating;
	private String userReviewMsge;
	private int hotelId;
	private int roomId;


}
